frame/lending/src/lib.rs

```diff
@@ -1046,6 +1033,31 @@ pub mod pallet {
 			market: MarketConfigOf<T>,
 			market_account: &T::AccountId,
 		) -> Result<(), DispatchError> {
+			let borrow_asset = T::Vault::asset_id(&market.borrow_asset_vault)?;
+
+			// Check is price actual yet
```

Comment:
may you do separate function/method for this code?

Reply:
done